cluster/gce/gci/configure-helper.sh

```diff
@@ -1027,7 +1046,12 @@ function start-kube-proxy {
   fi
   local container_env=""
   if [[ -n "${ENABLE_CACHE_MUTATION_DETECTOR:-}" ]]; then
-    container_env="env:\n    - name: KUBE_CACHE_MUTATION_DETECTOR\n    value: \"${ENABLE_CACHE_MUTATION_DETECTOR}\""
+    # Daemonset manifest needs a different indent.
```

Comment:
Ahh, I see above you introduced is_daemonset_manifest var to check the second argument of prepare-kube-proxy-manifest-variables() 

This is very error-prone, and hard to maintain in a long run.